cronjobs/calculate_daily_activity.py

```
@@ -0,0 +1,55 @@
+        logger.info(f"day {day} was already saved")
+        return
+
+    save_day(day)
+    check_and_save(day - timedelta(days=1))
+
+
+def day_was_saved(activity):
+    if len(activity) > 0:
+        return True
+
+    return False
```

Ruff

Return the condition `len(activity) > 0` directly

Replace with `return len(activity) > 0`

(SIM103)